spec/browse_cook_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "rails_helper"
+
+feature "browse cook" do
+  scenario "successfully" do
+    cook = Cook.create!(
+      name: "TEST_NAME",
+      cuisine: "TEST_CUISINE",
+      city: "TEST_CITY",
+    )
+
+    visit cook_path(cook)
+
+    expect(page).to have_content "TEST_NAME"
+    expect(page).to have_content "TEST_CUISINE"
+    expect(page).to have_content "TEST_CITY"
+  end
+
+  scenario "and edit successfully" do
+    cook = Cook.create!(
+      name: "TEST_NAME",
+      cuisine: "TEST_CUISINE",
+      city: "TEST_CITY",
+    )
```

Comment:
Is it worth adding FactoryBot to reduce duplication?

Reply:
Yes, it will be great to get your help on this next time we pair. Thank you